lib/SimpleSAML/XML/Shib13/AuthnResponse.php

```diff
@@ -260,9 +273,9 @@ public function getIssuer() {
 		$nodelist = $this->doXPathQuery($query);
 
 		if ($attr = $nodelist->item(0)) {
-			return $attr->value;
+			return $attr->nodeValue;
```

Comment:
Yes, but it's my fault. I change it because I though it is a \DOMElement but it is a \DOMAttr. I tested it and it works fine because internally must be equivalent (both extends from DOMNode). I'm going to revert the change.